src/utils/vscodeUtils.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as fse from 'fs-extra';
+import * as vscode from 'vscode';
+
+const outputChannel = vscode.window.createOutputChannel("Azure CosmosDB");
+
+export function getOutputChannel(): vscode.OutputChannel {
+    return outputChannel;
+}
+
+export interface IDisposable {
+    dispose(): void;
+}
+
+export function dispose<T extends IDisposable>(disposables: T[]): T[] {
```

Comment:
I realize this is a file rename, so this line of code may not be down to you. 

This [documentation page](https://vshaxe.github.io/vscode-extern/vscode/Disposable.html) mentions a void return.  Any reason why we're returning an array here?